packages/components/buttons/accessible-button/src/accessible-button.js

```diff
@@ -9,12 +9,16 @@ import getNormalizedButtonStyles from './accessible-button.styles';
 
 const propsToOmit = ['onClick'];
 
+const getIsEnterOrSpace = e =>
+  e.key === ' ' || e.key === 'Spacebar' || e.which === 13 || e.which === 32;
```

Comment:
What is `' '`? :). Is there maybe like a itty-bity util library we can use for this? Or do you think it's fine inlining it?

Reply:
`' '` is for the Spacebar.

I just double-checked a bit deeper and simplified this line, according to this spec that React uses for the `key` field of their synthetic events: https://www.w3.org/TR/uievents-key/#keys-whitespace